src/main/java/org/veupathdb/service/eda/ss/model/reducer/ancestor/AncestorExpander.java

```diff
@@ -71,7 +72,7 @@ private void setCurrentIfNotStarted() {
    */
   private Long nextMatch() {
     // Check if current entity equals ancestor.
-    if (currentDescendant.getValue() != currentEntity) {
+    if (!Objects.equals(currentDescendant.getValue(), currentEntity)) {
```

Comment:
why are we using Objects.equals here?

Reply:
This is a fix for a warning for using `==` on `Long`s